.github/workflows/build.yml

```
@@ -202,35 +199,56 @@ jobs:
           EOF
           )
           docker run -v $(pwd):/src \
             quay.io/pypa/manylinux_2_34_x86_64:latest \
             /bin/bash -c "$script"
-      - uses: actions/upload-artifact@v4
+      - uses: actions/upload-artifact@v6
         with:
           name: pip-wheel-${{ matrix.python }}
           path: wheelhouse/*
           if-no-files-found: error
+  build-any-wheel:
```

actionlint (1.7.11)

[error] job "build-any-wheel" needs job "lint" which does not exist in this workflow

(job-needs)